backend/api/projects.py

```diff
@@ -327,18 +310,17 @@ def update_comment(project_id, comment_id):
     """
     try:
         manifest = g.projects.find_one({'_id': project_id})
-        if manifest is None:
+        if not manifest:
             raise ApiException("Project not found", 404)
         if 'text/plain' not in request.content_type:
             raise ApiException("Content-Type header must include 'text/plain'", 400)
 
         if not manifest['comments']:
             raise ApiException("Project has no comments", 404)
-        # TODO Usermanagement
-        comment_msg = request.data.decode('utf-8')
+
         for comment in manifest['comments']:
-            if str(comment_id) == str(comment['id']):
-                comment['message'] = comment_msg
+            if str(comment_id) == str(comment['id']) and is_permitted(current_user, comment):
```

Comment:
shouldn't a user be able to delete or edit any comments under his projects, even if they aren't his own?